packages/cli/src/UserManagement/UserManagementHelper.ts

```diff
@@ -46,3 +47,14 @@ export function sanitizeUser(user: User): PublicUser {
 	const { password, resetPasswordToken, createdAt, updatedAt, ...sanitizedUser } = user;
 	return sanitizedUser;
 }
+
+/**
+ * Check if the endpoint is `POST /users/:id`.
+ */
+export function isPostUsersId(req: express.Request, restEndpoint: string): boolean {
+	return (
+		req.method === 'POST' &&
+		new RegExp(`/${restEndpoint}/users/[\\w\\d-]*`).test(req.url) &&
+		!req.url.includes('reinvite')
```

Comment:
Added L58 check since `/users/:id/reinvite` requires auth - the old regex left it unprotected otherwise.

Reply:
this is currently in the middleware of the feature branch